std_msgs/Header header
uint32 sensor_id
float64 mag_x
float64 mag_y
float64 mag_z
geometry_msgs/Pose sensor_pose
